tutorials/warmstart/configs/pre_training_config.yaml

```
@@ -240,23 +225,16 @@ gradient_clipper:
 
 progress_subscriber:
   component_key: progress_subscriber
-  variant_key: rich
-  config:
-    global_rank: ${settings.cuda_env.global_rank}
-    num_seen_steps: ${settings.training_progress.num_seen_steps}
-    num_target_steps: ${settings.training_target.num_target_steps}
-    train_dataloader_tag: ${train_dataloader.config.dataloader_tag}
-    eval_dataloaders:
-      instance_key: eval_dataloaders
-      pass_type: BY_REFERENCE
+  variant_key: dummy
+  config: {}
 
 evaluation_subscriber:
   component_key: results_subscriber
   variant_key: wandb
   config:
     global_rank: ${settings.cuda_env.global_rank}
     project: warmstart_tutorial
-    mode: ONLINE
+    mode: OFFLINE
     experiment_id: ${settings.experiment_id}
     directory: wandb_storage
```

```suggestion
    directory: ../data/wandb_storage
```
The temporary W&B logs currently end up in the folder `tutorials/warmstart/scripts` and will not be removed after the tests. With the changes, they will end up in the temporary `data` folder (along with checkpoints and tokenized data) instead, which is automatically removed.